src/e2e/scala/temple/SimpleE2ETest.scala

```diff
@@ -24,8 +23,8 @@ class SimpleE2ETest extends FlatSpec with Matchers {
       ),
     )
 
-    // Two folders should have been generated
-    val expectedFolders = Set("templeuser-db", "templeuser").map(dir => basePath.resolve(dir))
+    // Four folders should have been generated
```

Comment:
```suggestion
    // Exactly these folders should have been generated
```

Prevents them getting out of sync, and clarifies that it’s not just the number of folders, we are checking which ones are generated